VEHICLE/SetVehicleExclusiveDriver.md

```diff
@@ -1,25 +1,18 @@
 ---
 ns: VEHICLE
-aliases: ["0x41062318F23ED854"]
+aliases: ["0xB5C51B5502E85E83", "_SET_VEHICLE_EXCLUSIVE_DRIVER_2"]
 ---
 ## SET_VEHICLE_EXCLUSIVE_DRIVER
 
 ```c
-// 0x41062318F23ED854 0xFF62D324
-void SET_VEHICLE_EXCLUSIVE_DRIVER(Vehicle vehicle, cs_type(Ped) BOOL toggle);
+// 0xB5C51B5502E85E83 0xAA8BD440
```

Comment:
Again, not sure if changing hashes is properly supported.
Maybe @blattersturm could verify this?